config/config.example.yml

```diff
@@ -49,39 +29,15 @@ services:
     workspace_dir: ${WORKSPACE_DIR}
 
 # [Required] This section defines how to synchronize permissions between services when they share resources
-# This is used only for the synchronization of permissions between Magpie services/resources.
 sync_permissions:
   # Friendly name to identify a sync point (The value is not used by Cowbird so this can be any relevant keyword)
   user_workspace:
-    # [Required] This section defines a list of services and resources that exists in Magpie.
-    # For more info on the services available in Magpie :
-    #     https://pavics-magpie.readthedocs.io/en/latest/services.html#available-services
-    #     https://pavics-magpie.readthedocs.io/en/latest/autoapi/magpie/services/index.html
```

Comment:
Quick references like this should be kept here. 
In the official doc, they can be reported using `.. seealso::` bloc with corresponding references. The reference can be better formatted with a title as well.